src/_locales/en/messages.json

```diff
@@ -346,5 +346,29 @@
     "welcome_1b_dragging": "Drag your key backup into this dialog…",
     "welcome_1b_file_selected": "Selected exported key backup: {{filename}}",
     "untitled_wallet": "Untitled Wallet",
-    "your_message": "Your message…"
+    "your_message": "Your message…",
+    "plugin_file_service_display_name": "Plugin: File Service",
+    "plugin_file_service_file_key_copied": "File key copied.",
+    "plugin_file_service_on_insert": "Use the file",
+    "plugin_file_service_uploading_preparing": "Preparing",
+    "plugin_file_service_uploading_estimating_time": "Estimating time…",
+    "plugin_file_service_uploading_in_minute_remaining": "{{seconds}}s remaining",
+    "plugin_file_service_uploading_remaining": "{{minutes}}m {{seconds}}s remaining",
+    "plugin_file_service_uploading_on_cancal": "Uploading… Please do not close this page.",
+    "plugin_file_service_show_more": "Show More",
+    "plugin_file_service_file_key": "File Key:",
+    "plugin_file_service_unencrypted": "This file is not encrypted",
+    "plugin_file_service_signing_failed": "Remote signing failed.",
+    "plugin_file_service_on_encrypt_it": "Make It Encrypted",
```

Comment:
I remember that in a previous discussion, we decided that "Encrypt it" is better than "Make It Encrypted".